src/Microsoft.DotNet.Build.CloudTestTasks/WriteItemsToJson.cs

```diff
@@ -3,15 +3,16 @@
 // See the LICENSE file in the project root for more information.
```

Comment:
You can probably revert this whole file. None of the changes here look substantive.